extractors/python_functions/isbn_extraction/code_snippet.md

```diff
@@ -1,11 +1,10 @@
 ```python
 import re
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "isbn"
 
-def isbn_ext(record) -> List[Tuple[str, int, int]]:
+def isbn_ext(record):
```

Comment:
ext

Reply:
Because all the modules for the endpoint have the suffix "_extraction" I would say we keep and apply the same naming schema to the source code as well!